src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Figure implements AreaCalculator, Drawable {
+    private String color;
```

Comment:
As far as you already have `Colors` enum, feel free to use it as a type of color variable. Also, I would say it is more preferable to use singular rather than plural: `Color`